quickstarts/mux-video/config.yml

```diff
@@ -0,0 +1,35 @@
+slug: mux-video
+description: |
+  ## What is Mux?
+
+  Mux is video infrastructure for the internet. With just a few lines of code, companies can get real-time analytics at world-leading scale, and understand their streaming data, monitor user engagement, and improve the quality of experience for viewers. Mux is trusted by today’s largest broadcasters and leaders in streaming video to reach their goals with actionable insights. And together with Mux Video, developers can also build video into their products quickly, reliably, and at global scale.
+  Learn more at [mux.com/data](https://mux.com/data)
+
+
+  ### Get started!
+
+  The Mux - New Relic integration utilizes Amazon Kinesis Firehose to get data into New Relic. 
+
+  Check out the [documentation](https://docs.newrelic.com/docs/logs/forward-logs/mux-video-streaming-firehose/) to learn more about setting up New Relic monitoring for of data from Mux.
+summary: |
+  Mux is video infrastructure that makes it easy for development teams to get actionable data at industry-leading scale, and build video into their products quickly and reliably.
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic (Joseph Counts)
+title: Mux Video Analytics (via Amazon Firehose)
+installPlans:
```

Comment:
we are not longer allowing installPlans. Please delete the install plan you made and this reference to installPlans.

Reply:
done.